ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -491,19 +585,43 @@ private DeleteResult run_delete_v2(String accountName, String containerName, Str
     }
     // only need to issue an update statement if the row was not already marked as deleted.
     if (!alreadyDeleted) {
-      try (PreparedStatement statement = connection.prepareStatement(SOFT_DELETE_QUERY_V2)) {
-        // use the current time
-        statement.setTimestamp(1, currentTime);
-        statement.setInt(2, accountId);
-        statement.setInt(3, containerId);
-        statement.setString(4, blobName);
-        statement.setLong(5, version);
-        statement.executeUpdate();
-      }
+      applySoftDelete(accountId, containerId, blobName, version, currentTime, connection);
```

Comment:
nit: I suggest probably we can rename deletionTime to originalDeletionTime and rename currentTime to currentDeleteTime. At the first glance, this currentTime is a little bit confused to me.

Reply:
Done